pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -75,57 +75,45 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *servingv1alpha1
 		}
 	}
 
-	manifest, err := common.InstalledManifest(original)
-	if err != nil {
-		logger.Error("Unable to fetch installed manifest, some resources may not be finalized", err)
-		manifest, err = common.TargetManifest(original)
-	}
-	if err != nil {
-		logger.Error("Unable to fetch target manifest, no cluster-scoped resources will be finalized", err)
+	logger.Info("Deleting cluster-scoped resources")
+	// Create new, empty manifest with valid client and logger
+	manifest := r.manifest.Append()
+
+	// TODO: add ingress, etc
+	stages := common.Stages{common.AppendInstalled, r.transform}
```

Comment:
I find this one strange in particular. I don't think of these as reconcilation stages at all. The suggestion above would clear that out though.

Reply:
I don't think of them as reconciliation stages either. I specifically removed the word "reconcile" out of the name to avoid this confusion. But they are stages in the construction of the manifest, and I expect there will be more, e.g. `ingress`